lightning/src/util/config.rs

```diff
@@ -213,6 +213,11 @@ pub struct ChannelHandshakeConfig {
 	/// back to a `anchors_zero_fee_htlc` (if [`Self::negotiate_anchors_zero_fee_htlc_tx`]
 	/// is set) or `static_remote_key` channel.
 	///
+	/// To have confidence that a force-close transaction will reach miners and get confirmed,
+	/// zero-fee commitment channels require your Bitcoin node **and some of its peers** to
```

Comment:
Might be worth being explicit that there needs to be a path between your node and miners running 29+.